astropy/modeling/fitting.py

```diff
@@ -897,7 +897,7 @@ def _fitter_to_model_params(model, fps):
     # better to change this at some point
     if has_tied:
         for idx, name in enumerate(model.param_names):
-            if model.tied[name] != False:
+            if model.tied[name]:
```

Comment:
Similar to my comment about  removing `== True` above, perhaps it is safer to do `is not False` here.